cmd/mesh/manifest-migrate.go

```diff
@@ -59,7 +59,7 @@ func manifestMigrateCmd(rootArgs *rootArgs, mmArgs *manifestMigrateArgs) *cobra.
 }
 
 func valueFileFilter(path string) bool {
-	return filepath.Base(path) == "values.yaml"
+	return filepath.Ext(path) == YAMLSuffix && strings.HasPrefix(filepath.Base(path), "values")
```

Comment:
i'm concerned this is not going to be robust. we're relying on a filename being of the form values*.yaml - is there a better way to check for this, like file position wrt. charts root?

Reply:
I checked the helm package implementation, it just check whether the file name equals "values.yaml", like what was implemented here before. That would work for installer files, but I am thinking the requirement should not be too strict because we have seen "values*.yaml" naming.

I am not sure whether it is necessary to validate the file schema or content here, instead I think it might be more clear to just give user message what files are being translated or not.